resource-managers/kubernetes/docker-minimal-bundle/src/main/docker/driver-py/Dockerfile

```diff
@@ -39,10 +39,10 @@ ENV PYSPARK_DRIVER_PYTHON python
 ENV PYTHONPATH ${SPARK_HOME}/python/:${SPARK_HOME}/python/lib/py4j-0.10.4-src.zip:${PYTHONPATH}
 
 CMD SPARK_CLASSPATH="${SPARK_HOME}/jars/*" && \
+    env | grep SPARK_JAVA_OPT_ | sed 's/[^=]*=\(.*\)/\1/g' > /tmp/java_opts.txt && \
```

Comment:
Piping to `readarray` didn't work for me; I believe the result was an empty array in the environment variable. But perhaps I was writing it incorrectly? I had this:

`env | grep SPARK_JAVA_OPT_ | sed 's/[^=]*=\(.*\)/\1/g' | readarray -t SPARK_DRIVER_JAVA_OPTS`